docs/Developers/Git_Basics.rst

```diff
@@ -166,6 +208,12 @@ shift + Z twice).
 Pushing the commit
 ------------------
 
+Before you push the commit, ensure that you are not developing on master again.
+Make sure that you are not developing on master by running:
```

Comment:
this line is pretty much a duplicate of the previous one. I think you can cut it and move the colon up.